clayui.com/src/utils/getSection.js

```diff
@@ -56,6 +60,10 @@ const toSectionElements = (
 };
 
 const toSectionItem = (item, paths) => {
+	if (item.isFormFolder) {
+		item.isFolder = true;
+	}
+
```

Comment:
I think we can remove `isFormFolder` from here and work only with the `navigationParent` information to create the grouping, you can try to do the following.

- Sweep the items' Array and create an object by grouping the corresponding paths to use later
  ```js
  {
     'path/some': ['path/children', '']
  }
  ```
- Sweep the elements to group, look for the path corresponding to the object created previously, and add all the corresponding elements of that path
  ```js
  elements
	.map((item, index, arr) => {
		const path = Object.keys(groups).find(path => path === item.slug);

		if (path) {
			return {
				...item,
				items: groups[path].map(path => arr.find(item => item.slug === path))
			}
		}

		return item;
	})
  ```
- Sweep elements to filter and remove any elements with `navigationParent`

I have not tested these examples above and it is also likely that you will have to do something else.